workflow.rst

```diff
@@ -158,13 +158,20 @@ As configured, the following property is used by the marking store::
 
     The marking store type could be "multiple_state" or "single_state". A single
     state marking store does not support a model being on multiple places at the
-    same time.
+    same time. This means a "workflow" must use a "multiple_state" marking store
```

Comment:
Are you sure Workflow **must** use a multiple marking store?

Reply:
Short answer: yes
Long answer: no, but it's **way too** dangerous. I forbid that in SF 4.* and now it's not possible to choose :) you can read this: https://symfony.com/blog/new-in-symfony-4-3-workflow-improvements#simpler-configuration to get more information
So I want to make this very clear and easy to choose (or to not choose actually) in 3.4

Reply:
Thank you for the answer and the link. 

Great job!

I agree with this change.